OPAL/tac/src/main/scala/org/opalj/tac/fpcf/analyses/alias/package.scala

```diff
@@ -0,0 +1,41 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj
+package tac
+package fpcf
+package analyses
+
+import org.opalj.ai.ValueOrigin
+import org.opalj.br.PC
+import org.opalj.br.fpcf.properties.Context
+import org.opalj.br.fpcf.properties.alias.PersistentUVar
+import org.opalj.tac.DUVar
+import org.opalj.value.ValueInformation
+
+package object alias {
+
+    type V = DUVar[ValueInformation]
+
+    type AllocationSite = (Context, PC)
+
+    /**
+     * Converts the given [[UVar]] into an equivalent [[PersistentUVar]].
+     *
+     * @param uVar The UVar to convert.
+     * @param stmts The statements of the method the UVar is defined in.
+     * @return A persistent UVar for the given UVar.
+     */
+    final def persistentUVar(uVar: UVar[ValueInformation])(implicit stmts: Array[Stmt[V]]): PersistentUVar = {
```

Comment:
This doesn't look like it should be part of this package object (and neither does the next method). Aren't they already available elsewhere? If not, place them in a more general place.